onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphTransformer.h

```diff
@@ -20,18 +19,18 @@
     public:
         GraphTransformer(
             const std::string& name, 
-            const onnxruntime::IExecutionProvider* provider
-        );
+            const onnxruntime::InlinedHashSet<std::string_view>& compatible_execution_providers = {}
```

Comment:
>const onnxruntime::InlinedHashSet<std::string_view>& compatible_execution_providers = {} [](http://example.com/codeflow?start=12&length=88)

This seems unnecessary since the transform is meant to be specific to the DML EP

Reply:
Yeah, makes sense. I will remove it. I was also in dilemma about it, but then I thought it would be good to pass the compatible ep list just for the sanity.